### PR TITLE
EZP-24619: Extract Solr search engine into a separate repository

### PR DESCRIPTION
**Pull request by pspanja**

### CODE REVIEW

**Comment by pspanja** [comment]
@lolautruche as discussed on Skype, separated autoload namespaces for better performance in https://github.com/ezsystems/ezplatform-solr-search-engine/commit/55deed61612cc2749f49f130af70bb9c71ab621e.

File: bundle/EzPublishSolrSearchEngineBundle/DependencyInjection/EzPublishSolrSearchEngineExtension.php [hunk 1]
```diff
@@ -62,9 +62,8 @@ public function load(array $configs, ContainerBuilder $container)
         // Loading configuration from Core/settings
         $loader = new YamlFileLoader(
             $container,
-            new FileLocator(__DIR__ . '/../../../Publish/Core/settings')
+            new FileLocator(__DIR__ . '/../../../src/eZ/Publish/Core/settings')
```
Comment (lolautruche) [inline bundle/EzPublishSolrSearchEngineBundle/DependencyInjection/EzPublishSolrSearchEngineExtension.php]: Needs to be `lib/`

Comment (pspanja) [inline bundle/EzPublishSolrSearchEngineBundle/DependencyInjection/EzPublishSolrSearchEngineExtension.php]: Fixed all instances and rebased.

**Comment by pspanja** [comment]
@bdunogier changed `tests/` structure as discussed in https://github.com/ezsystems/ezplatform-solr-search-engine/commit/aeda35fc9bf096372ddcd258cf5e094112253f6f.

**Comment by pspanja** [comment]
@bdunogier changed `tests/` structure as discussed in https://github.com/ezsystems/ezplatform-solr-search-engine/commit/d809aa1b9050605a71c27a23fb8f145ee5a30346.

**Comment by bdunogier** [comment]
+1, looks good to me.

**Comment by andrerom** [comment]
+1

**Pull request merged**